homeassistant/components/device_tracker/owntracks.py

```diff
@@ -85,7 +89,12 @@ def owntracks_event_update(topic, payload, qos):
         if data['event'] == 'enter':
             zone = hass.states.get("zone.{}".format(location))
             with LOCK:
-                if zone is not None:
+                if zone is None:
```

Comment:
With this current approach, when you get a location for a beacon, will you still change the location of a user too? Isn't that… unexpected?

I thought that a new location for a person would update the location for a beacon, unless people leave the beacon behind ?